grobid-core/src/main/java/org/grobid/core/jni/JEPThreadPool.java

```diff
@@ -128,7 +128,7 @@ private Jep createJEPInstance() {
      * The method will return to the thread its dedicated Jep instance
      * (or create one the first time).
      */
-    public Jep getJEPInstance() {
+    public synchronized Jep getJEPInstance() {
```

Comment:
I realised it should really be synchronized. Otherwise it could try to initialise two instances at the same time. One could use concurrent checks in a way that doesn't require synchronization. But not sure if it's worth it.